src/Components/ArticleWithTags/ArticleWithTags.js

```diff
@@ -0,0 +1,4 @@
+import elementsForArticle from '../../HOC/HOC.js';
+import { Article } from '../Article/Article.js';
+
+export default elementsForArticle('withTags')(Article); 
```

Comment:
this incorrect using HOC. 
for example, I want to use Article with Tags and with Remarks. How can I implemented this with current approach?

correct hoc must to do next `const ArticleWithAllFunctionals = withRemarks(withTags(Article)))`